NEWS.md

```diff
@@ -1,3 +1,7 @@
+# dv.clinlines 1.0.4
+
+* Adapt basic_info parameter to adhere module standard
```

Comment:
```suggestion
* To comply with the DaVinci module standards, the list entries expected by the `basic_info` module argument were updated as follows: `subject_level_dataset_name` instead of `data`, `trt_start_var` instead of `trt_start`, `trt_end_var` instead of `trt_end`, and `icf_date_var` instead of `icf_date`.
```